07_payment_channels.asciidoc

```diff
@@ -29,32 +29,32 @@ Lightning is Bitcoin. It's just a different way of using the Bitcoin system.
 
 Before we understand payment channels, we have to take a small step back and understand how ownership and control work in Bitcoin.
 
-When someone says they "own" bitcoin they typically mean that they know the private key of a Bitcoin address that has some unspent transaction outputs (see <<bitcoin_fundamentals_review>>). The private key allows them to sign a transaction to spend that bitcoin by transferring it to a different address. In Bitcoin "ownership" of bitcoin can be defined as the _ability to spend_ that bitcoin.
+When someone says they "own" bitcoin they typically mean that they know the private key of a Bitcoin address that has some unspent transaction outputs (see <<bitcoin_fundamentals_review>>). The private key allows them to sign a transaction to spend that bitcoin by transferring it to a different address. In Bitcoin, "ownership" of bitcoin can be defined as the _ability to spend_ that bitcoin.
 
-Keep in mind that the term "ownership" as used in Bitcoin is distinct from the term "ownership" used in a legal sense. A thief who has the private keys and can spend Bitcoin is a _de-facto owner_ of that Bitcoin even though they are not a lawful owner.
+Keep in mind that the term "ownership" as used in Bitcoin is distinct from the term "ownership" used in a legal sense. A thief who has the private keys and can spend bitcoin is a _de-facto owner_ of that bitcoin even though they are not a lawful owner.
 
 [TIP]
 ====
-Bitcoin ownership is only about control of keys and the ability to spend the Bitcoin with those keys. As the popular Bitcoin saying by goes: "Your keys, your coins - not your keys, not your coins"
+Bitcoin ownership is only about control of keys and the ability to spend the bitcoin with those keys. As the popular saying goes: "Your keys, your coins - not your keys, not your coins."
 ====
 
 ==== Diversity of (independent) ownership and multisig
 
-Ownership and control of private keys is not always in the hands of one person. That's where things get interesting and complicated. We know that more than one person can come to know the same private key, either through theft or because the original holder of the key makes a copy and gives it to someone else. Are all these people owners? In a practical sense they are, because any one of the people who know the private key can spend the bitcoin without the approval of any other.
+Ownership and control of private keys is not always in the hands of one person. That's where things get interesting and complicated. We know that more than one person can come to know the same private key, either through theft or because the original holder of the key makes a copy and gives it to someone else. Are all these people owners? In a practical sense they are, because any one of the people who knows the private key can spend the bitcoin without the approval of any other.
```

Comment:
I don't think your change is grammatically correct

Reply:
Thanks for the feedback.

I do think my version is grammatically correct. However, it does sound wrong because of the way it is written.  Compare:

“anyone knows ....”

“any one of them knows”

“any one of the people knows”

The subject of the phrase is “one” and NOT “people”.  To the ear my correction sounds wrong because we are putting “people” next to “knows” - “people knows“ - which is an unusual combination.

Anyway, this may all be academic. In one of his Youtube videos Andreas announced that the book has now been sent to the publishers.  I don’t believe further changes to the version on Github are being made.

Zendo102